lib/Zend/XmlRpc/Value.php

```diff
@@ -486,13 +486,15 @@ protected static function _createSimpleXMLElement(&$xml)
      */
     protected static function _extractTypeAndValue(SimpleXMLElement $xml, &$type, &$value)
     {
-        list($type, $value) = each($xml);
+        $value = reset($xml);
```

Comment:
technically this is a logic change, as `each()` returns the current, and `reset()` the first.
Also `each()` moves the internal coursor.
But from the usage here it seems to not matter, and also makes the result more predictable, so Iam ok with leaving it as it is.